packages/hint-html-checker/src/hint.ts

```diff
@@ -85,7 +86,7 @@ export default class HtmlCheckerHint implements IHint {
             lastLine: number;
             hiliteStart: number;
             message: string;
-            subType: string;
+            subType: keyof typeof Severity;
```

Comment:
what does `keyof typeof` do in TypeScript?

Reply:
https://github.com/Microsoft/TypeScript/issues/14106#issuecomment-280253269